cli/graphql.md

```diff
@@ -1756,6 +1756,81 @@ patterns and best practices baked in but it also provides number of escape hatch
 those situations where you might need a bit more control. Here are a few common use cases
 you might find useful.
 
+#### Filter Subscriptions by model fields and/or relations
+
+In real world use cases, particularly in multi-tenant scenarios, subscribed clients may not always want to receive every change for a model type. These are useful features for limiting the objects that are returned by a client subscription. It is crucial to remember that subscriptions can only filter by what fields are returned from the mutation query. Keep in mind, these two methods can be used together to create truly robust filtering options.
```

Comment:
Please remove "In a real world use case" as this can be subjective for different customers